service/k8sclient/kubectl/kubectl.go

```diff
@@ -32,6 +35,11 @@ import (
 	"github.com/percona-platform/dbaas-controller/utils/logger"
 )
 
+const (
+	defaultKubectl = "dbaas-kubectl-1.16"
```

Comment:
That's good, but let's temporary set it to `"minikube kubectl --"` with a FIXME comment